src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetCoreMvc2Integration.cs

```diff
@@ -266,7 +273,14 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
             catch (Exception ex)
             {
                 // profiled app will continue working as expected without this method
-                Log.ErrorException($"Error resolving {methodDef}", ex);
+                Log.ErrorRetrievingMethod(
+                    exception: ex,
+                    moduleVersionPointer: moduleVersionPtr,
+                    mdToken: mdToken,
+                    opCode: opCode,
+                    instrumentedType: DiagnosticSourceTypeName,
+                    methodName: nameof(AfterAction),
+                    instanceType: diagnosticSource?.GetType().AssemblyQualifiedName);
```

Comment:
We're instrumenting a static method, so there's no instance in this case.

Reply:
Same as above, the info is possibly valuable, even though it's not semantically in line with the term "instance"